### PR TITLE
Currency selector

### PR DESCRIPTION
**Pull request by ggarcia852**

### CODE REVIEW

File: src/components/CoinData/CoinData.js [hunk 1]
```diff
@@ -44,8 +45,21 @@ export default class CoinData extends React.Component {
     }
   };
 
+  setCurrency = () => {
+    let currency = this.props.currency
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinData/CoinData.js:52]: can use const

File: src/components/CoinData/CoinData.js [hunk 2]
```diff
@@ -44,8 +45,21 @@ export default class CoinData extends React.Component {
...
+    this.setState({currency})
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinData/CoinData.js:53]: this.setState({ currency })

File: src/components/CurrencySelector/CurrencySelector.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    currency: "usd"
+  }
+
+  handleSelect = (e) => {
+    this.setState({ currency: e.target.value })
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (this.state.currency !== prevState.currency){
+      this.setState({currency: this.state.currency})
```
Comment (yoloOnTheBattlefield) [inline src/components/CurrencySelector/CurrencySelector.js:16]: what's the purpose of this, state.currency is already state.currency

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/CurrencySelector/CurrencySelector.js (hunk 3 above)
Comment (ggarcia852) [reply src/components/CurrencySelector/CurrencySelector.js:16]: ah right, got rid of the setState

**Pull request merged**